dart/dynamics/Joint.h

```diff
@@ -347,6 +354,16 @@ class Joint : public virtual common::Subject
   /// Get upper limit for position
   virtual double getPositionUpperLimit(size_t _index) const = 0;
 
+  /// Gets whether a generalized coordinate is cyclic. Returns true if and only
```

Comment:
Personally, for classes I always explicitly state the class name, so the subject isn't left to be inferred. But yes, if the subject is being implied by the sentence, and the subject is a class or struct, then having the ``s`` would be necessary.